lib/molecules/admin-bar/template.html

```diff
@@ -0,0 +1,25 @@
+<div class="toolbar" role="navigation" aria-label="Toolbar" tabindex="0" ng-if="vm.adminBarEnabled">
```

Comment:
I think is fine to have these styles on ng-web-app, just in case we have to customize them on each app.